apps/web-giddh/src/app/shared/send-email-invoice/send-email-invoice.component.scss

```diff
@@ -0,0 +1,5 @@
+@import "../../../assets/css/color.scss";
+
+.modal-header{
+    background-color: $zambezi;
```

Comment:
modal header color should be dark blue check once it on the browser for make sure

Reply:
done